### PR TITLE
DAT-4565: Mock ads for anon naviagates to RWA from menu test

### PR DESCRIPTION
**Pull request by lukaszkonieczny**
Ad covers whole screen so it's not possible to click menu entry point

cc @Wikia/west-wing @drets @PiotrGackowski @ludwikkazmierczak

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/RecentWikiActivityTest.java [hunk 1]
```diff
@@ -18,6 +18,7 @@
 public class RecentWikiActivityTest extends NewTestTemplate {
 
   @Test(groups = "mercury_recentWikiActivity_anonNavigateToRWAfromMenu")
+  @Execute(mockAds = "true")
```
Comment (SebastianMarzjan) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/RecentWikiActivityTest.java]: I know that this is the interstitial, so as long as that ad behavior is verified differently, LGTM

Comment (nandy-andy) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/RecentWikiActivityTest.java]: Good point, @SebastianMarzjan! :) We've got [tests for interstitial](https://github.com/Wikia/selenium-tests/blob/master/src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsInterstitial.java#L42) the most important cases for us are tested it. However, we won't mind if somebody adds to it test checking if the close button works :japanese_ogre:

Comment (lukaszkonieczny) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/RecentWikiActivityTest.java]: @Wikia/adeng @SebastianMarzjan tests for interstitial are now expanded on verifying does close button work.

**Comment by SebastianMarzjan** [comment]
LGTM. 
@nandy-andy @lukaszkonieczny - While I usually prefer to have a single assertion per test, it looks like in this specific example any of the assertions failing would be a sign of a big problem, and splitting the assertions would increase test execution time significantly.

**Pull request merged**